dav1d-sys/src/lib.rs

```diff
@@ -3,7 +3,664 @@
+    pub width: [::std::os::raw::c_int; 2usize],
+    pub height: ::std::os::raw::c_int,
+    pub frame_offset: ::std::os::raw::c_int,
+    pub temporal_id: ::std::os::raw::c_int,
+    pub spatial_id: ::std::os::raw::c_int,
+    pub show_existing_frame: ::std::os::raw::c_int,
+    pub existing_frame_idx: ::std::os::raw::c_int,
+    pub frame_id: ::std::os::raw::c_int,
+    pub frame_presentation_delay: ::std::os::raw::c_int,
+    pub show_frame: ::std::os::raw::c_int,
+    pub showable_frame: ::std::os::raw::c_int,
+    pub error_resilient_mode: ::std::os::raw::c_int,
+    pub disable_cdf_update: ::std::os::raw::c_int,
+    pub allow_screen_content_tools: ::std::os::raw::c_int,
+    pub force_integer_mv: ::std::os::raw::c_int,
+    pub frame_size_override: ::std::os::raw::c_int,
+    pub primary_ref_frame: ::std::os::raw::c_int,
+    pub buffer_removal_time_present: ::std::os::raw::c_int,
+    pub operating_points: [Dav1dFrameHeader_Dav1dFrameHeaderOperatingPoint; 32usize],
+    pub refresh_frame_flags: ::std::os::raw::c_int,
+    pub render_width: ::std::os::raw::c_int,
+    pub render_height: ::std::os::raw::c_int,
+    pub super_res: Dav1dFrameHeader__bindgen_ty_2,
+    pub have_render_size: ::std::os::raw::c_int,
+    pub allow_intrabc: ::std::os::raw::c_int,
+    pub frame_ref_short_signaling: ::std::os::raw::c_int,
+    pub refidx: [::std::os::raw::c_int; 7usize],
+    pub hp: ::std::os::raw::c_int,
+    pub subpel_filter_mode: Dav1dFilterMode,
+    pub switchable_motion_mode: ::std::os::raw::c_int,
+    pub use_ref_frame_mvs: ::std::os::raw::c_int,
+    pub refresh_context: ::std::os::raw::c_int,
+    pub tiling: Dav1dFrameHeader__bindgen_ty_3,
+    pub quant: Dav1dFrameHeader__bindgen_ty_4,
+    pub segmentation: Dav1dFrameHeader__bindgen_ty_5,
+    pub delta: Dav1dFrameHeader__bindgen_ty_6,
+    pub all_lossless: ::std::os::raw::c_int,
+    pub loopfilter: Dav1dFrameHeader__bindgen_ty_7,
+    pub cdef: Dav1dFrameHeader__bindgen_ty_8,
+    pub restoration: Dav1dFrameHeader__bindgen_ty_9,
+    pub txfm_mode: Dav1dTxfmMode,
+    pub switchable_comp_refs: ::std::os::raw::c_int,
+    pub skip_mode_allowed: ::std::os::raw::c_int,
+    pub skip_mode_enabled: ::std::os::raw::c_int,
+    pub skip_mode_refs: [::std::os::raw::c_int; 2usize],
+    pub warp_motion: ::std::os::raw::c_int,
+    pub reduced_txtp_set: ::std::os::raw::c_int,
+    pub gmv: [Dav1dWarpedMotionParams; 7usize],
+}
+
+#[repr(C)]
+#[derive(Debug, Copy, Clone)]
+pub struct Dav1dFrameHeader__bindgen_ty_1 {
+    pub data: Dav1dFilmGrainData,
+    pub present: ::std::os::raw::c_int,
+    pub update: ::std::os::raw::c_int,
+}
+
+#[repr(C)]
+#[derive(Debug, Copy, Clone)]
+pub struct Dav1dFrameHeader_Dav1dFrameHeaderOperatingPoint {
+    pub buffer_removal_time: ::std::os::raw::c_int,
+}
+
+#[repr(C)]
+#[derive(Debug, Copy, Clone)]
+pub struct Dav1dFrameHeader__bindgen_ty_2 {
+    pub width_scale_denominator: ::std::os::raw::c_int,
+    pub enabled: ::std::os::raw::c_int,
```

Comment:
Will add `use std::os::raw::c_int` etc at the top so there's less line-noise everywhere

Reply:
use `core::ffi:c_int` maybe?

Reply:
I used `std::os::raw` here because otherwise we'd depend on Rust >= 1.64 just for that. If that's OK for you I can change it accordingly.